docs/source/en/guides/cli.md

```diff
@@ -0,0 +1,445 @@
+    download            Download files from the Hub
+    lfs-enable-largefiles
+                        Configure your repository to enable upload of files > 5GB.
+    lfs-multipart-upload
+                        Command will get called by git-lfs, do not call it directly.
+    scan-cache          Scan cache directory.
+    delete-cache        Delete revisions from the cache directory.
+
+options:
+  -h, --help            show this help message and exit
+```
+
+If the CLI is correctly installed, you should see a list of all the options available in the CLI. If you get an error message such as `command not found: huggingface-cli`, please refer to the [Installation](../installation) guide. 
+
+<Tip>
+
+The option `--help` we used above is very convenient to get more details about a command. You can use it at any time if you want to list all options available and get their details. For example, `huggingface-cli upload --help` provides more information on how to upload files using the CLI.
+
+</Tip>
+
+## huggingface-cli login
+
+In a lot of cases, you must be logged in to a Hugging Face account to interact with the Hub: download private repos, upload files, create PRs,... To do so, you need a [User Access Token](https://huggingface.co/docs/hub/security-tokens) from your [Settings page](https://huggingface.co/settings/tokens). The User Access Token is used to authenticate your identity to the Hub.
+
+Once you have your token, run the following command in your terminal:
+
+```bash
+>>> huggingface-cli login
+```
+
+This command will prompt you for a token. Copy-paste yours and press *Enter*. Then you'll be ask if the token should also be saved as a git credential. Press *Enter* again (default to yes) if you plan to use `git` locally. Finally, it will call the Hub to check that your token is valid and save it locally.
+
+```
+_|    _|  _|    _|    _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|_|_|_|    _|_|      _|_|_|  _|_|_|_|
+_|    _|  _|    _|  _|        _|          _|    _|_|    _|  _|            _|        _|    _|  _|        _|
+_|_|_|_|  _|    _|  _|  _|_|  _|  _|_|    _|    _|  _|  _|  _|  _|_|      _|_|_|    _|_|_|_|  _|        _|_|_|
+_|    _|  _|    _|  _|    _|  _|    _|    _|    _|    _|_|  _|    _|      _|        _|    _|  _|        _|
+_|    _|    _|_|      _|_|_|    _|_|_|  _|_|_|  _|      _|    _|_|_|      _|        _|    _|    _|_|_|  _|_|_|_|
+
+To login, `huggingface_hub` requires a token generated from https://huggingface.co/settings/tokens .
+Token: 
+Add token as git credential? (Y/n) 
+Token is valid (permission: write).
+Your token has been saved in your configured git credential helpers (store).
+Your token has been saved to /home/wauplin/.cache/huggingface/token
+Login successful
+```
+
+Alternatively if you want to login without being prompt, you can pass the token from the command line. It is recommended to use an environment variable when doing so to avoid having your token pasted in your command history.
```

Comment:
```suggestion
Alternatively, if you want to log-in without being prompted, you can pass the token directly from the command line. To be more secure, we recommend passing your token as an environment variable to avoid pasting it in your command history.
```